src/core/engine/processors/abstract_processor.py

```diff
@@ -13,11 +13,18 @@ def __init__(self, app_processor_config, app_context=None):
         self.destroyed = False
         self.last_execution_time = datetime.now().timestamp()
 
+    @abstractmethod
+    def init(self):
+        pass
+
     @abstractmethod
     def process(self, delta_time):
         pass
 
     def run(self):
+
+        self.init()
+
         if self.app_processor_config.on_start:
             self.app_processor_config.on_start(
```

Comment:
What do you think about using this instead of init?

Reply:
done.